src/services/helpdesk/hooks/index.js

```diff
@@ -14,9 +15,27 @@ Deine ${data.cloud}
 	`;
 }
 
+async function generateSystemInformation(account) {
+	const { userId, username } = account;
+
+	const userInformation = await userModel.userModel.findOne(userId)
```

Comment:
please use the user modelservice instead.

Reply:
Changed it to modelservice now